roles/imagemagick/tasks/main.yml

```diff
@@ -1,14 +1,30 @@
 ---
 # ROLE: imagemagick
-# roles/imagemagick/tasks/main.yml 
+# roles/imagemagick/tasks/main.yml
 #
 # USAGE:
-# Installs the most recent release of ImageMagick, if you need a specific major or minor release, set the `imagemagick_ver:` variable
-#   EG:
-#     - { role: imagemagick, imagemagick_ver: '6.6' }   # will install the highest version matching the 6.6 prefix
+# Installs the most recent release of ImageMagick, if you need a
+# specific major or minor release, set the `imagemagick_ver:` variable
 
 - include: im_libraries.yml
-- include: openjpg.yml
+
+- stat: path=/usr/local/lib/libtiff.a
+  register: tiff_stat
```

Comment:
Should I use the full version—in libtiff's case `5.2.4`, or just the major version (5)?